src/citadel/citadel.utils.ts

```diff
@@ -230,21 +235,65 @@ export async function getCitadelKnightingRoundsStats(): Promise<CitadelKnightsRo
   const client = new GraphQLClient(CITADEL_KNIGHTS_ROUND_SUBGRAPH_URL);
   const graphSdk = setKnightsRoundStatsSdk(client);
 
-  const knightsRoundStatResp = await graphSdk.KnightsRounds();
+  let totalVotes: VoteFragment[] = [];
+
+  let voteId;
+  while (true) {
+    const voteData = await graphSdk.Votes({
+      first: 2,
```

Comment:
will remove this subsequntly just want to verify iteration in staging w/ low vote counts